react-toggle-switch-component/my-app/src/ToggleButton.jsx

```diff
@@ -0,0 +1,23 @@
+import './ToggleButton.css';
+import { useState } from 'react';
+
+export default function ToggleButton() {
+  const [isOn, setIsOn] = useState(false);
+  let text = '';
```

Comment:
I like using the ternary operator to set this all in one line of code. It also lets me declare the variable `const`:
`const text = isOn? 'On' : 'Off';`